salesforce/events.go

```diff
@@ -189,6 +189,7 @@ func (c *Connector) CreateEventRelayConfig(
 // nolint: lll
 // https://developer.salesforce.com/docs/atlas.en-us.api_tooling.meta/api_tooling/tooling_api_objects_eventrelayconfig.htm?q=EventRelayConfig
 func (c *Connector) RunEventRelay(ctx context.Context, cfg *EventRelayConfig) error {
+	// TODO should there be a BaseURL?
 	location, err := joinURLPath("tooling/sobjects/EventRelayConfig", cfg.Id)
```

Comment:
This could be a constant and be updated to use getURL / getDomainURL

Reply:
I changed URL methods such that:

getRestApiURL -> rest prefix
getDomainURL -> url starting from base domain
getURIPartEventRelayConfig -> returns relative path to `tooling/sobjects/EventRelayConfig`